xml/templates/civicrm_state_province.tpl

```diff
@@ -3619,7 +3619,7 @@ INSERT INTO civicrm_state_province (id, country_id, abbreviation, name) VALUES
 (5166, 1075, "TR", "Tampere Region"),
 (5167, 1075, "OB", "Ostrobothnia"),
 (5168, 1075, "NK", "North Karelia"),
-(5169, 1075, "NO", "Nothern Ostrobothnia"),
+(5169, 1075, "NO", "Northern Ostrobothnia"),
```

Comment:
It might be a good idea but it would be a separate task to do... Or is it a blocker to merge?